lookout/style/typos/preprocessing/print_frequencies.py

```diff
@@ -0,0 +1,30 @@
+import lzma
+from typing import Set
+
+import numpy
+import pandas
+
+from lookout.style.typos.utils import COLUMNS
+
+
+def print_frequencies(tokens_set: Set[str], frequency_column: str, id_stats: pandas.DataFrame,
+                      path: str) -> None:
+    """
+    Dump frequencies of tokens from tokens_set to a file.
+
+    Frequencies info is obtained from id_stats dataframe.
+    """
+    frequencies = {}
+    for token, row in id_stats.iterrows():
+        if token in tokens_set:
+            frequencies[token] = row[frequency_column]
+    with lzma.open(path, "wt") as f:
```

Comment:
Too many formats: pickled pandas, numpy array-s, ad-hoc archives with lists. Just write the CSV to stdout, the compression can always be done later using a pipe. Use `import csv` and make the delimiter configurable.